.github/workflows/pimcore-demo.yml

```diff
@@ -103,4 +120,24 @@ jobs:
            echo "Install failed, skipping build"
            exit 1;
         fi
-        docker-compose down -v --remove-orphans
+
+    - name: Test pimcore upgrade
+      if: ${{ matrix.pimcore_upgrade_version != '' }}
+      env:
+          UPGRADE_VERSION: "${{ matrix.pimcore_upgrade_version }}"
+      run: |
+        # Echo commands and terminate on first error
+        set -ex
+        cd sample-project/
+        docker-compose exec -T -- php composer require pimcore/pimcore:"${UPGRADE_VERSION}"
```

Comment:
One last thing: we need to execute command `./bin/console doctrine:migrations:migrate` to run migrations after upgrade as they do not run automatically.